github-actions/scan/__test__/integrationtest/01-start.sh

```diff
@@ -113,15 +113,15 @@ fi
 
 ## generate SecHub test certificate
 if [ ! -f $SERVER_CERTFILE_PATH ]; then
-    mkdir "${SERVER_FOLDER_PATH}" -p
+    mkdir -p "${SERVER_FOLDER_PATH}"
     keytool -genkey -alias tomcat -storetype PKCS12 -keyalg RSA -keysize 2048 -keystore "${SERVER_CERTFILE_PATH}" -validity 3650 -storepass 123456 --dname "CN=localhost, OU=ID"
 fi
 
 
 # Start SecHub async
 SECHUB_LOGFILE=$SERVER_FOLDER_PATH/server.log
 
-rm $SECHUB_LOGFILE -f # remove old log files on start
+rm -f $SECHUB_LOGFILE # remove old log files on start
```

Comment:
suggest to sanitize the path string
```suggestion
rm -f "$SECHUB_LOGFILE" # remove old log files on start
```